ssl/ssl_lib.c

```diff
@@ -3473,6 +3473,68 @@ char *SSL_get_shared_ciphers(const SSL *s, char *buf, int size)
     return buf;
 }
 
+/** Specify the EC point formats to be used by default by the SSL_CTX */
+__owur int SSL_CTX_set_ec_point_formats(SSL_CTX *ctx, const unsigned char *formats,
+                                        size_t formats_len)
+{
+    unsigned char *ecpointformats;
+    size_t i;
+
+    if (formats == NULL || formats_len == 0)
+        return 0;
+
+    /* Check all formats are within our recognized values: */
+    for (i = 0; i < formats_len; i++) {
+        if (formats[i] < TLSEXT_ECPOINTFORMAT_first ||
+            formats[i] > TLSEXT_ECPOINTFORMAT_last)
+            return 0;
```

Comment:
Calling `ERR_raise` here to give the caller a clue as to what went wrong would not be a bad idea.